docs/components/modeler/bpmn/user-tasks/user-tasks.md

```diff
@@ -35,12 +35,13 @@ This can be used to define which user the task can be assigned to. One or both o
 attributes can be specified simultaneously:
 
 - `assignee`: Specifies the user assigned to the task. [Tasklist](/components/tasklist/introduction-to-tasklist.md) will claim the task for this user.
+- `candidateUsers`: Specifies the users that the task can be assigned to.
 - `candidateGroups`: Specifies the groups of users that the task can be assigned to.
 
-Typically, the assignee and candidate groups are defined as static values (e.g. `some_username` and
+Typically, the assignee, candidate users and candidate groups are defined as static values (e.g. `some_username`, `some_username, another_username` and
```

Comment:
Just one minor comment/nit pick here -- need an Oxford comma, so:

"Typically, the assignee, candidate users**,** and candidate groups are defined as static values..."